force-app/main/default/classes/SubmitCompensations_Batch_Test.cls

```diff
@@ -0,0 +1,24 @@
+@isTest
+public class SubmitCompensations_Batch_Test {
+
+    @isTest
+    public static void testIt(){
+        List<Compensation__c> comp_list = new List<Compensation__c>();
+        Compensation__c comp1 = new Compensation__c();
+        comp1.RecordTypeId = '0123h000000H5smAAC';
```

Comment:
Sebas, I would suggest you to create a data factory class to generate the records to test against instead of "hard coding it", its a best practice =)

Reply:
Ok, i will do it for the next time thanks.